libraries/freertos_plus/standard/freertos_plus_tcp.cmake

```diff
@@ -0,0 +1,40 @@
+afr_module(NAME "freertos_plus_tcp" INTERNAL)
+
+set(src_dir "${CMAKE_CURRENT_LIST_DIR}/freertos_plus_tcp")
+set(inc_dir "${CMAKE_CURRENT_LIST_DIR}/freertos_plus_tcp/include")
+set(test_dir "${CMAKE_CURRENT_LIST_DIR}")
```

Comment:
The value of test_dir does not seem correct based on what it was earlier.

Reply:
Updated